test/quantization/test_qat.py

```diff
@@ -98,12 +98,15 @@
 )
 from torchao.utils import (
     _is_fbgemm_gpu_genai_available,
+    get_current_accelerator_device,
     is_fbcode,
     is_sm_at_least_89,
 )
 
 # TODO: put this in a common test utils file
 _CUDA_IS_AVAILABLE = torch.cuda.is_available()
+_GPU_IS_AVAILABLE = torch.accelerator.is_available()
+_DEVICE = get_current_accelerator_device()
```

Comment:
Directly use "_DEVICE  is None" in the following changes? No need to define _GPU_IS_AVAILABLE.